packages/pwa-buildpack/envVarDefinitions.json

```diff
@@ -165,7 +165,7 @@
                     "name": "CHECKOUT_BRAINTREE_TOKEN",
                     "type": "str",
                     "desc": "Specify a Braintree API token token to direct the Venia storefront to communicate with your Braintree instance. You can find this value in Braintree's Control Panel under Settings > API Keys > Tokenization Keys.",
-                    "default": "sandbox_8yrzsvtm_s2bg8fs563crhqzk"
+                    "example": "sandbox_8yrzsvtm_s2bg8fs563crhqzk"
```

Comment:
```suggestion
                    "example": "sandbox_8yrzsvtm_s2bg8fs563crhqzk",
                    "default": ""
```

Reply:
I modeled this one after `MAGENTO_BACKEND_URL`: 

```json
            "name": "Connecting to a Magento store",
            "variables": [
                {
                    "name": "MAGENTO_BACKEND_URL",
                    "type": "url",
                    "desc": "Connect to an instance of Magento 2.3 by specifying its public domain name.",
                    "example": "https://master-7rqtwti-mfwmkrjfqvbjk.us-4.magentosite.cloud/"
                }
            ]
```